packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -95,15 +95,18 @@ export async function importBlock(
   this.logger.verbose("Added block to forkchoice and state cache", {slot: blockSlot, root: blockRootHex});
 
   // We want to import block asap so call all event handler in the next event loop
-  setTimeout(() => {
+  setTimeout(async () => {
     this.emitter.emit(routes.events.EventType.block, {
       block: blockRootHex,
       slot: blockSlot,
       executionOptimistic: blockSummary != null && isOptimisticBlock(blockSummary),
     });
 
     if (blockInput.type === BlockInputType.postDeneb) {
```

Comment:
throw error if it's not `BlockInputType.postDeneb` to assert that we already have full blobs at this phase

Reply:
thrown this assertion on L72